Makefile

```
@@ -44,10 +44,23 @@ test: demo-certs
 test-json: demo-certs
 	@go test -v -json $(GOTEST_ARGS) ./...
 
-# Run tests with coverage
+# Run tests with coverage and fail if it slips below the threshold. Without a
+# gate the number only ever drifts down, one untested branch at a time.
+COVERAGE_THRESHOLD ?= 80.0
+
 .PHONY: test-coverage
 test-coverage: demo-certs
```

checkmake (0.3.2)

[warning] Target body for "test-coverage" exceeds allowed length of 5 lines (11).

(maxbodylength)